### PR TITLE
DLWP scoring fixes

### PR DESCRIPTION
**Pull request by ktangsali**
# Earth-2 MIP Pull Request

## Description
Fixes the bug in the time shift code in dlwp wrapper. Also adds more comments to the scoring notebook for better clarity.

Closes https://github.com/NVIDIA/earth2mip/issues/140

## Checklist

- [x] I am familiar with the Contributing Guidelines.
- [x] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.
- [ ] The CHANGELOG.md is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/earth2mip/issues) is linked to this pull request.

### CODE REVIEW

File: examples/03_model_scoring.py [hunk 1]
```diff
@@ -137,18 +137,24 @@
 # can then be pose process using some of the utility functions Earth-2 MIP provides.
 
 # %%
+import numpy as np
 from earth2mip.inference_medium_range import save_scores, time_average_metrics
 
-time = datetime.datetime(2017, 1, 1, 0)
-initial_times = [time + datetime.timedelta(days=30 * i) for i in range(12)]
+# Use 52 initializations.
```
Comment (NickGeneva) [inline examples/03_model_scoring.py:143]: For the example, please reduce this to at most 12. Designed to just be a sampler, not a full blown analysis

File: examples/03_model_scoring.py [hunk 2]
```diff
@@ -137,18 +137,24 @@
...
+]  # randomly select a time between 00, 06, 12 and 18 hrs.
 
 # Output directoy
 output_dir = "outputs/03_model_scoring"
 if not os.path.exists(output_dir):
     os.makedirs(output_dir, exist_ok=True)
     output = save_scores(
         model,
-        n=20,  # 12 hour timesteps
+        n=56,  # 6 hour timesteps (14-day forecast)
```
Comment (NickGeneva) [inline examples/03_model_scoring.py:157]: Can we make this just a 7 day forecast?

File: examples/03_model_scoring.py (hunk 1 above)
Comment (ktangsali) [reply examples/03_model_scoring.py]: Updated, and added more docs

File: examples/03_model_scoring.py (hunk 2 above)
Comment (ktangsali) [reply examples/03_model_scoring.py]: Made it 7 day, and added some commentary on how the n relates to forecast length.

File: earth2mip/networks/dlwp.py [hunk 3]
```diff
@@ -25,12 +25,95 @@
...
+        """Yield (time, unnormalized data, restart) tuples
+
+        restart = (time, unnormalized data)
+        """
+        if restart:
+            raise NotImplementedError("Restart capability not implemented.")
+        # do not implement restart capability
+        restart_data = None
+
+        # print(x.shape)
```
Comment (NickGeneva) [inline earth2mip/networks/dlwp.py:97]: can delete this

Comment (ktangsali) [reply earth2mip/networks/dlwp.py]: Done

**Comment by NickGeneva** [comment]
Merged here: https://github.com/NVIDIA/earth2mip/pull/146

Thanks for fixing this!

**Pull request closed without merging**